### DISCUSSION TITLE
Why doesn't Bootstrap have Datepicker component up to now?

### DISCUSSION THREAD

**Discussion by julianjupiter** [twbs/bootstrap#39186] [Ideas]
I'm wondering why up to now Bootstrap is not adding Datepicker component? It seems that BS is on the only framework that does not have Datepicker.

**Comment by patkon** [comment]
That is an interesting question. Do we still need a date picker at all in 2023? It's [provided by the browser](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) anyway, isn't it?

In my projects I often use [Tempus Dominus](https://getdatepicker.com) and wonder now if it is still needed at all.

**Reply by julianjupiter** [reply]
Yes. HTML5 has now input types for date & time and so are other input types. It is interesting that other input types and other form elements have existed way ahead of date and time and are easier to style yet BS bothered to add components for them. It is also interesting that most if not all UI frameworks have components for date, time and calendar.

**Comment by dazam1204** [comment]
> That is an interesting question. Do we still need a date picker at all in 2023? It's [provided by the browser](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) anyway, isn't it?
> 
> In my projects I often use [Tempus Dominus](https://getdatepicker.com) and wonder now if it is still needed at all.

It makes sense the moment you are developing a dashboard and you need to use date picker to get a specific range of data.
In any case, is there any solution other than browser native? Or maybe understand how to customize the native one, could be possible?
